srcpkgs/luarocks/template

```diff
@@ -18,6 +18,10 @@ license="MIT"
 homepage="https://luarocks.org/"
 distfiles="https://luarocks.org/releases/luarocks-${version}.tar.gz"
 checksum=1236a307ca5c556c4fed9fdbd35a7e0e80ccf063024becc8c3bf212f37ff0edf
+alternatives="
+	luarocks:luarocks:/usr/bin/luarocks-5.3
+	luarocks:luarocks-admin:/usr/bin/luarocks-admin-5.3
+"
```

Comment:
put the doublequote on the end of the previous line

Reply:
Sigh. There's always something.
I have half a mind to go hack on xlint, especially since the repo is inconsistent on a lot of these minor formatting issues.

Reply:
And indent with one space instead of tabs.

Reply:
Oh, so *that's* why. I originally (incorrectly) had two spaces:

```
$ xlint srcpkgs/luarocks/template
srcpkgs/luarocks/template:22: indent with tabs
```